app/models/report.rb

```diff
@@ -63,8 +63,8 @@ def statuses
     Status.with_discarded.where(id: status_ids)
   end
 
-  def media_attachments
-    MediaAttachment.where(status_id: status_ids)
+  def media_attachments_count
+    statuses.pluck(:ordered_media_attachment_ids).compact.sum(&:size)
```

Comment:
Would be 0 for all statuses older than the application of this PR.

Maybe something like the following?

```suggestion
  def media_attachments_count
    statuses_to_query = []
    count = 0

    statuses.pluck(:id, :ordered_media_attachment_ids).each do |id, ordered_ids|
      if ordered_ids.nil?
        statuses_to_query << id
      else
        count += ordered_ids.size
      end
    end

    count += MediaAttachment.where(status_id: statuses_to_query).count unless statuses_to_query.empty?
    count
```